tests/test_edgeql_net_schema.py

```diff
@@ -66,7 +66,7 @@ async def test_edgeql_net_http_request_response_01(self):
         await self.assert_query_result(
             '''
             with
-                nh as module net::http,
+                nh as module std::net::http,
```

Comment:
I did have to fully qualify this.

Reply:
Yeah, we should indeed fix WITH to accept unqualified names.